content/en/docs/latest/Automation/Karaoke_Templater/Tutorial_1.md

```diff
@@ -39,7 +39,7 @@ lyrics yourself, or get someone else to do it. You can learn how to time
 karaoke at the [Karaoke_Timing_Tutorial]({{< relref "Karaoke_Timing_Tutorial" >}}).
 
 We'll call song lyrics with basic karaoke timing (\\k timing), but no other
-special effects, for _timed karaoke_. When an effect has been applied to the
+special effects, as _timed karaoke_. When an effect has been applied to the
```

Comment:
"call […] as \<something\>" is only a slight improvement imo. Just remove it entirely, i.e.
```suggestion
special effects, _timed karaoke_. When an effect has been applied to the
```